From: Simon McVittie <smcv@debian.org>
Date: Thu, 8 Jun 2017 22:38:31 +0100
Subject: Install pkg-config metadata to ${datadir}

Because this package's only development files are the D-Bus
introspection XML files and it does not ship a library, its
development files can in fact be architecture-independent.
pkg-config searches ${datadir}/pkgconfig in addition to
${libdir}/pkgconfig in order to support such packages.

Signed-off-by: Simon McVittie <smcv@debian.org>
---
 Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 2642e83..6fbafe0 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -31,7 +31,7 @@ lib_LTLIBRARIES =
 include data/Makefile.am.inc
 include src/Makefile.am.inc
 
-pkgconfigdir = $(libdir)/pkgconfig
+pkgconfigdir = $(datadir)/pkgconfig
 pkgconfig_DATA = xdg-desktop-portal.pc
 EXTRA_DIST += xdg-desktop-portal.pc.in
 
